__tests__/app.test.js

```diff
@@ -128,6 +129,47 @@ describe("GET /api/articles", ()=>{
         })
     })
 
+
+describe("GET /api/articles/:article_id/comments", ()=>{
+    test('200: responds with an array of comments for the given article_id', ()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({body})=>{
+            const {comments} = body
+            expect(comments).toBeSortedBy("created_at")
+            comments.forEach((comment)=>{
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: 1
+                })
+            })
+
+        })
+    })
+    test('404: responds with an error message for a not found path', ()=>{ return request(app)
```

Comment:
A bit of weird formatting, I'd put the return on it's own line to keep it consistent and looking clean